API Testing/pillar2-external-test-stub/99-qa/uktr/Nil return - Valid.bru

```diff
@@ -1,10 +1,10 @@
 meta {
-  name: Nil return - Valid
+  name: Amend to Nil return - Valid
   type: http
-  seq: 6
+  seq: 3
 }
 
-post {
+put {
```

Comment:
Do we need to keep a test for POSTing nil returns around? I absolutely do not know the business usecases here, so I'm not sure if that's a valuable test or not.